CLA.md

```diff
@@ -0,0 +1,66 @@
+# `calibr8` CLA
+
+Thank you for your interest in contributing to Forschungszentrum Jülich GmbH - Institute for Bio- and Geosciences: IBG-1's `calibr8` project ("We" or "Us").
+
+The purpose of this contributor agreement is to clarify and document the rights granted by contributors to Us.
```

Comment:
Is it normal to capitalise Us?

Reply:
The reason is probably to make clear this is not a colloquial _us_ but the _Us_ as defined earlier